libraries/chain/include/cyberway/genesis/ee_genesis_container.hpp

```diff
@@ -40,10 +40,18 @@ struct section_ee_header {
 };
 
 struct vote_ee_object {
+    account_name voter;
     int16_t weight;
     uint64_t time;
 };
 
+struct reblog_ee_object {
+    account_name account;
+    string title;
+    string body;
+    uint64_t time;
+};
+
 struct message_ee_object {
```

Comment:
message_ee_object should contain additional fields:
`string parent_author_username;`  - `parent_author` in `user@domain` format
`string author_username;`  - `author` in `user@domain` format

Reply:
#586